docs/netobserv_cli_flows_config.md

```diff
@@ -19,31 +19,32 @@ USER=netobserv make commands
 
 - The following table shows all supported features options.
 
-| Option           | Description                     | Possible values   | Default |
-|------------------|---------------------------------|-------------------|---------|
-| --enable_pktdrop | Enable packet drop              | true, false       | false   |
-| --enable_rtt     | Enable round trip time          | true, false       | false   |
-| --enable_dns     | Enable DNS tracking             | true, false       | false   |
-| --interfaces     | Interfaces to match on the flow | e.g., "eth0,eth1" | ""      |
+| Option           | Description                      | Possible values            | Default  |
+|------------------|----------------------------------|----------------------------|----------|
+| --enable_pktdrop | Enable packet drop               | true, false                | false    |
+| --enable_rtt     | Enable round trip time           | true, false                | false    |
+| --enable_dns     | Enable DNS tracking              | true, false                | false    |
+| --interfaces     | Interfaces to match on the flow  | e.g., "eth0,eth1"          | ""       |
+| --log-level      | Components logs                  | for example debug or trace | info     |
+| --max-time       | Maximum capture time             | for example 10m or 30s     | 5m       |
+| --max-bytes      | max-bytes: maximum capture bytes | for example 10000000 (1MB) | 50000000 |
```

Comment:
I'm merging https://github.com/netobserv/network-observability-cli/pull/65 so you don't have to manage this here ;)